### PR TITLE
v2v:add TC '-ip option is required when convert guest from VMware without vddk'

### PR DESCRIPTION
**Pull request by vwu-vera**
Please check bz#1960087

### CODE REVIEW

**Comment by vwu-vera** [comment]
(1/1) type_specific.io-github-autotest-libvirt.function_test_esx.negative_test.must_ip.esx_67.rhev.rhv.it_default: PASS (16.74 s)

**Comment by vwu-vera** [comment]
@xiaodwan Please help to review. Thanks. :)

File: v2v/tests/cfg/function_test_esx.cfg [hunk 1]
```diff
@@ -680,9 +680,18 @@
             only esx_65
             version_required = "[libvirt-7.8.0-1,)"
             main_vm = VM_NAME_ESX_SATA_DISK_V2V_EXAMPLE
+        - must_ip:
+            only esx_67
+            only it_default
+            only negative_test
+            version_required = "[virt-v2v-1.45.98-1.el9,)"
```
Comment (xiaodwan) [inline v2v/tests/cfg/function_test_esx.cfg:687]: When comparing the versions, we only care the real version numbers and ignore the rhel versions.
In this case, '[virt-v2v-1.45.98-1,)' is correct. In your code, if it runs in virt-v2v-1.45.98-1.el8 on rhel8, the case will be skipped unexpectedly.
Others look fine.

Comment (vwu-vera) [reply v2v/tests/cfg/function_test_esx.cfg]: Thanks for pointing out. Has been modified accordingly.

**Review by xiaodwan — APPROVED** [review]

**Pull request merged**